Multisig/UI/Transactions/TransactionsView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TransactionsView.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/27/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionsView: View {
+
+    @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
+    var selectedSafe: FetchedResults<Safe>
+
+    var body: some View {
+        let safe = selectedSafe.first
+        return ZStack {
+            Rectangle()
```

Comment:
I have a feeling that this zstack and background must be part of the `EmptyListPlaceholder,` not part of this view

Reply:
This way, when we define the content view we have to add the background again to it,
the background actually is for the list itself not for empty state only

Reply:
I see